tests/gpu_benchmark/modal_hashpow_gpu_v2.py

```
@@ -0,0 +1,192 @@
+        
+        println!("  Difficulty {}: {}", diff, time_str);
+    }
+}
+RUST
+
+source $HOME/.cargo/env
+rustc bench.rs -O -o bench
+./bench
+"""
+    
+    result = subprocess.run(
```

Ruff (0.14.14)

`subprocess` call: check for execution of untrusted input

(S603)